src/styles/main.scss

```diff
@@ -1,3 +1,64 @@
+$standart-month: 31;
+
+$month-length: 28;
```

Comment:
The $month-length variable should be removed. Instead, use the month-length modifier as a class with values 28, 29, 30, and 31 (use @for).